README.md

```diff
@@ -21,10 +21,55 @@ other components of OpenShift via support for a baremetal platform type.
 
 # Instructions
 
-## Configuration
+## Preparation
+
+Considering that this is a new install on a clean OS, the next tasks should be performed prior the installation:
+
+1. Enable passwordless sudo for the current user
+
+    `echo "$USER  ALL=(ALL) NOPASSWD: ALL" >/etc/sudoers.d/${USER}`
+
+2. In case of RHEL, invoke `subscription-manager` in order to `register` and `attach` the subscription
+
+3. Install new packages
+
+    `sudo dnf update`
+    `sudo dnf install git make wget tmux jq`
+
+4. Clone the dev-scripts repository
+
+    `git clone https://github.com/openshift-metal3/dev-scripts`
+
+5. Create a config file
 
-Make a copy of the `config_example.sh` to `config_$USER.sh`, and set the
-`PULL_SECRET` variable to the secret obtained from cloud.openshift.com.
+    `cp config_example.sh config_$USER.sh`
+
+6. Configure secrets:
+   1. Download personal pull-secret file from https://cloud.redhat.com/openshift/install/pull-secret
+   2. Get the shared secret from a coworker
```

Comment:
Can we document the api.ci steps?

Login to the web UI of https://api.ci.openshift.org/console/catalog, click on your name in the upper right corner, click "copy login command" and then after logging in, do `oc registry login --to=pull-secret.txt`

Reply:
This is the personal pull-secret, not the shared secret that Russell is in charge of. Am I wrong?

Reply:
The shared one has now been revoked.  These instructions should now instruct people to use their own.

Reply:
Thanks, exactly the same way?

Reply:
The instructions @stbenjam included here look right to me

Reply:
Just to combine all the steps in one place, this worked for me in the lab:

1. Download personal pull-secret file from https://cloud.redhat.com/openshift/install/pull-secret
2. Login to the web UI of https://api.ci.openshift.org/console/catalog click on your name in the upper right corner, click "copy login command" 
3. Run the login command from the previous step
4. Run “oc registry login --to=shared-secret.txt”
5. In config_stack.sh, first paste the contents of the pull-secret in the PULL_SECRET variable and append registry.svc.ci.openshift.org shared secret to it

Automating 5. would be nice as @dhellmann suggested however I the script may need updating to reflect the new way, it didn't work for me in it's current form.